packages/react-components/react-spinner/src/components/Spinner/Spinner.test.tsx

```diff
@@ -32,11 +32,11 @@ describe('Spinner', () => {
 
   it('doesnt render Spinner when slot is overridden', () => {
```

Comment:
I think the name of these tests should probably be changed.

Reply:
I think it'd make sense to keep the name and change the test to query for the SVG and verify it's not there